src/sentry/static/sentry/app/views/projectGeneralSettings.jsx

```diff
@@ -313,3 +327,34 @@ export default class ProjectGeneralSettings extends AsyncView {
     );
   }
 }
+
+const ProjectGeneralSettingsContainer = createReactClass({
+  mixins: [Reflux.listenTo(ProjectsStore, 'onProjectsUpdate')],
+  onProjectsUpdate(projects) {
+    if (!this.changedSlug) return;
```

Comment:
Style is a little inconsistent here compared to https://github.com/getsentry/sentry/blob/2e6d31e4ce32f16e9dabc4414a2592828de55c70/src/sentry/static/sentry/app/stores/projectsStore.jsx#L30-L33

Reply:
Fixed this in `projectsStore`